lib/chargebeex/client.ex

```diff
@@ -1,9 +1,24 @@
 defmodule Chargebeex.Client do
   @moduledoc false
-  def endpoint(path, params \\ %{}) do
-    namespace = Application.get_env(:chargebeex, :namespace)
-    host = Application.get_env(:chargebeex, :host)
-    base_path = Application.get_env(:chargebeex, :path)
+
+  defp config(site, path) do
+    default = Application.get_env(:chargebeex, path)
+
+    case site do
+      :default ->
+        default
+
+      site ->
+        :chargebeex
+        |> Application.get_env(site)
+        |> Keyword.get(path, default)
```

Comment:
Maybe we shouldn't make it fallback silently 🤔  If a `site` is given and the application doesn't behave as expected (because of the typo in the configuration for example) maybe it should raise an error?